sql/parser/sql.y

```diff
@@ -1392,13 +1392,13 @@ rename_stmt:
   {
     $$ = &RenameDatabase{Name: Name($3), NewName: Name($6)}
   }
-| ALTER TABLE relation_expr RENAME TO name
+| ALTER TABLE relation_expr RENAME TO relation_expr
```

Comment:
I think the `TO` name should be `qualified_name`, not `relation_expr`. The latter rule allows for `ONLY table_name` and `table_name *` which I do not think makes sense in the context of renaming a table.